src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Publish.targets

```diff
@@ -106,7 +106,10 @@ Copyright (c) .NET Foundation. All rights reserved.
                  ResourceName="CannotHaveSingleFileWithoutRuntimeIdentifier" />
     <NETSdkError Condition="'$(PublishSingleFile)' == 'true' and '$(UseAppHost)' != 'true'"
                  ResourceName="CannotHaveSingleFileWithoutAppHost" />
-
+    <NETSdkError Condition="'$(UseWindowsForms)' == 'true' and '$(PublishTrimmed)' == 'true' and '$(BuiltInComSupport)' != 'true'"
```

Comment:
This logic doesn't error or warn when winforms apps that are publishing trimmed have `BuiltInComSupport` set to true, is that intended?

Reply:
Yes, it is intended. We generally do have warn for all trimmed apps at the end and winforms will have this warning as well. We expect that all WinForm apps will run through this error first and then need to explicitly take an action (set property to true) to go into trimming.